apps/portal/src/components/resumes/comments/CommentsForm.tsx

```diff
@@ -39,7 +39,14 @@ export default function CommentsForm({
       skills: '',
     },
   });
-  const reviewCreateMutation = trpc.useMutation('resumes.reviews.user.create');
+
+  const trpcContext = trpc.useContext();
+  const reviewCreateMutation = trpc.useMutation('resumes.reviews.user.create', {
```

Comment:
@yangshun I followed the `Todos` example and triggered `invalidateQueries` here, then in `CommentsList` I'll fetch the comments. Based on the network logs the `create` is triggered but the query isn't

Reply:
I'm not entirely sure about the issue but I think might be due to the order of how the queries are executed. There's an [article](https://lightrun.com/answers/prisma-docs-clarify-when-prisma-queries-are-run-prismapromise-behavior) provided by @peironggg and a [stackoverflow](https://stackoverflow.com/questions/71194355/react-query-invalidatequery-not-working-to-update-users-list) solution. Tried it and the query seems to be triggered after the mutation!